doc/source/Dependencies.rst

```diff
@@ -0,0 +1,162 @@
+Dependencies
+================
+
+.. contents:: Contents
+    :depth: 2
+
+
+IMPROVER builds on the functionality provided by a range of open source
+libraries.
+
+Some of these libraries are widely used throughout the IMPROVER code, so are
+considered as required for IMPROVER as a whole.
+
+Other libraries are only used in specific parts of IMPROVER.
+These libraries are optional for IMPROVER as a whole, but are required to use
+related parts of IMPROVER.
+Optional installation of these libraries allows for smaller sized installations
+of IMPROVER and reduces conda environment dependency solving difficulties.
+
+Required
+-----------------
+
+cartopy
+~~~~~~~~~~~~~~~~~
+Cartopy is used for grid projections and coordinate transformations.
+
+https://scitools.org.uk/cartopy/docs/stable/
+
+
+cftime
+~~~~~~~~~~~~~~~~~
+cftime provides functions for handling time in NetCDF files according to the
+Climate and Forecast (CF) conventions.
+
+https://unidata.github.io/cftime/
+
+
+cf-units
```

Comment:
Do dependencies of dependencies need to be documented?
improver doesn't cf_units directly as far as my grep suggests - so it's an dependency of iris, not improver no?
Are there others here we needn't include?

Reply:
Oops.  @gavinevans rightly corrected me that improver does use `cf_units` so scratch my comment above :)